src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -303,7 +303,7 @@
         {% endif %}
     {%- endfor %}
 
-    {% if not form.methodRendered %}
+    {% if not form.methodRendered and form.parent is null %}
```

Comment:
I hesitated suggesting to add a `FormView::needsMethodToBeRendered` instead, which will do this check inside, as rendering the method input does only makes sense for a root form anyway and would avoid this issue.

Besides that, we already use `form.parent` in multiple places, so I don't think the related issue should be considered here.